src/explorer/trialApp/TrialAppTreeItem.ts

```diff
@@ -124,6 +140,18 @@ export class TrialAppTreeItem extends SiteTreeItemBase implements ISiteTreeItem
         }
         return super.compareChildrenImpl(item1, item2);
     }
+
+    private async isTrialAppExpired(): Promise<boolean> {
+        const trialAppContext: ITrialAppContext | undefined = ext.context.globalState.get(TrialAppContext);
```

Comment:
Note: Probably could set the context to expired here too in this PR: https://github.com/microsoft/vscode-azureappservice/pull/1679